src/components/oscal-utils/OSCALLinkUtils.js

```diff
@@ -61,3 +61,22 @@ export function guessExtensionFromHref(url) {
   }
   return detectedExtension;
 }
+
+/**
+ * Finds if a fragment contains a control group.
+ * NOTE: Control groupings will contain two letters, followed by a "-" and a positive integer
```

Comment:
Will they? This feels _very_ SP 800-53 specific.

Reply:
My bad. I should have rewritten this comment after making the below function support all control schemes. Will fix this.